src/styles/main.scss

```diff
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import "header", "navigation", "container", "card", "reset";
```

Comment:
The reset.scss should be added as the first one! The order is important, because it works as a reset for all the included styles from the browser.